cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -240,6 +248,22 @@ public Network convert() {
         return network;
     }
 
+    private void addControlArea(Context context, CgmesControlAreaAdder cgmesControlAreaAdder, PropertyBag tf) {
+        String controlAreaId = tf.getId("ControlArea");
+        String controlAreaName = tf.getLocal("controlAreaName");
+        String energyIdentCodeEic = tf.getLocal("energyIdentCodeEic");
+        double netInterchange = tf.asDouble("netInterchange");
+        CgmesControlArea cgmesControlArea = cgmesControlAreaAdder.newCgmesControlArea(controlAreaId, controlAreaName, energyIdentCodeEic, netInterchange);
```

Comment:
Okay, this is a bit confusing because reading the code, it seems like you may add the same control area several time (i know it is not true since it is a map in the implementation but the API does not transcribe it